Competition2017/src/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -26,14 +27,20 @@
 
     private double previousPositionInches;
     
+    private final PIDManager headingDrivePid;
+    private ContiguousHeading targetHeading;
+    public final double defaultPValue = 1/80d;
+
     private int onTargetCount = 0;
     
     @Inject
     public DriveForDistanceCommand(DriveSubsystem driveSubsystem, XPropertyManager propManager) {
         this.driveSubsystem = driveSubsystem;
         this.requires(driveSubsystem);
         this.travelManager = new PIDManager("Drive to position", propManager, 0.1, 0, 0, 0.5, -0.5);
-        
+
+        headingDrivePid = new PIDManager("HeadingModule", propManager, defaultPValue, 0, 0);
+        targetHeading = new ContiguousHeading();
         onTargetCountThresholdProp = propManager.createPersistentProperty("DrvToPos min stabilization time", 3);
```

Comment:
Most importantly, this should legitimately be time, instead of number of calls to `isFinished`.

Naming-wise, `Stopped duration for completion` would be a reasonable phrase.